packages/stopwatch/package.json

```
@@ -0,0 +1,52 @@
+{
+	"name": "@sapphire/stopwatch",
+	"version": "1.0.0",
```

```suggestion
	"version": "0.0.0",
```

Version bump to 1.0.0 will be done by Lerna on first publish